Improve TC visibility

cristian-ungureanu opened the pull request:
### Summary
- Added the card in neve dashboard
- Changed the Notification react component to accept just a text + a link, until now it was only working only with plugins and themes updates

### Will affect visual aspect of the product
NO

### Test instructions
- The card should only appear if the White label is deactivated. Test if this is true.
- The link on the card should say "Discover Templates Cloud" if you don't have the Agency plan with an URL to Neve pricing and should say "Learn how to use Template Cloud" with a link to [documentation](https://docs.themeisle.com/article/1091-starter-sites-available-for-import#my-library)

After you test this, you need to have a look at [this PR too](https://github.com/Codeinwp/templates-patterns-collection/pull/79).


Closes Codeinwp/neve-pro-addon#1286.

tudor-themeisle commented:
The card is present even if the White Label is on -  https://prnt.sc/11ia4ec , https://prnt.sc/11ia5d8.
When I returned on the Welcome tab in Neve options, it appeared by itself with the White Label turned on https://prnt.sc/11ia87h .
I have the Agency plan, but I don't see a link on the card that is saying "Discover Templates Cloud".

cristian-ungureanu commented:
Sorry, Tudor, I was not clear enough when providing instructions. The white label module is considered enabled only if you have at least one field ( Theme name, plugin name, agency, etc.)

The link on the card should say "Discover Templates Cloud" if you don't have the Agency plan with an URL to Neve pricing and should say "Learn how to use Template Cloud" with a link to the documentation.

cc. @irinelenache

irinelenache commented:
@cristian-ungureanu the card still appears with Whitelabel on https://prnt.sc/11x183n

cristian-ungureanu commented:
@irinelenache thanks for testing. That's not the card that I added. I added this one https://prnt.sc/11x44pk. As you can see in your screenshot, the text in that box says "Irinel's theme ... " so it means that it's the desired behavior for that card to be present.

irinelenache commented:
@cristian-ungureanu sorry, i was misunderstanding the card with the notification, so everything works fine 🚀
